includes/class-wc-stripe-webhook-handler.php

```diff
@@ -125,6 +135,33 @@ public function check_for_webhook() {
 		}
 	}
 
+	/**
+	 * Check if the Stripe account has duplicate webhooks setup for this site.
+	 *
+	 * @since 9.1.0
+	 */
+	public function has_duplicate_webhooks_setup() {
+		$webhook_url = WC_Stripe_Helper::get_webhook_url();
+		$webhooks    = WC_Stripe_API::retrieve( 'webhook_endpoints' );
+
+		if ( is_wp_error( $webhooks ) || ! isset( $webhooks->data ) || empty( $webhooks->data ) ) {
+			return false;
+		}
+
+		$number_of_duplicate_webhooks = count( $webhooks );
```

Comment:
Shouldn't this start at `0` since we only want to count it if the webhook URL matches the site?

Nit: Maybe `$number_of_webhooks` instead of `$number_of_duplicate_webhooks`?

Reply:
I don't remember when I changed it to `count` from `0`. It makes the logic totally wrong here 😳

Reply:
Fixed this in [d26e5be](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3681/commits/d26e5be5aac8102f94a2a459637a4f4cd6b6a221). 

> Sorry, caught this late in the logs:
```
[02-Jan-2025 18:54:18 UTC] PHP Warning:  count(): Parameter must be an array or an object that implements Countable in /var/www/html/wp-content/plugins/woocommerce-gateway-stripe/includes/class-wc-stripe-webhook-handler.php on line 151
```

The change should also fix the above error as we don't have the usage of `count` now.